apps/web/app/api/v1/management/surveys/[surveyId]/route.ts

```
@@ -60,27 +62,44 @@ export const PUT = async (
       ...survey,
       ...surveyUpdate,
     });
+
     if (!inputValidation.success) {
       return responses.badRequestResponse(
         "Fields are missing or incorrectly formatted",
         transformErrorToDetails(inputValidation.error)
       );
     }
+
+    if (surveyUpdate.followUps && surveyUpdate.followUps.length) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)